packages/terra-collapsible-menu-view/src/CollapsibleMenuViewDivider.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import Menu from 'terra-menu';
+import 'terra-base/lib/baseStyles';
+import styles from './CollapsibleMenuView.scss';
+
+const cx = classNames.bind(styles);
+
+const contextTypes = {
+  isCollapsibleMenuItem: PropTypes.bool,
+};
+
+const CollapsibleMenuViewDivider = (props, { isCollapsibleMenuItem }) => {
+  if (isCollapsibleMenuItem) {
+    return <Menu.Divider />;
+  }
+  return <div className={cx(['divider', 'face-up-item'])} />;
```

Comment:
Should these menus spread custom props?

Reply:
I didn't have custom props on the Menu.Divider either, my thought on this was that the divider is purely a visual element. We don't have custom attributes for the dividers in other components. The only reason this is a component is because it seemed to be the easiest way for consumers to indicate where the items needed to be divided.